go/teams/chain.go

```diff
@@ -1439,6 +1439,18 @@ func (t *TeamSigChainPlayer) addInnerLink(
 		res.newState = prevState.DeepCopy()
 		err = t.parseTeamSettings(team.Settings, &res.newState)
 		return res, err
+	case libkb.LinkTypeDeleteRoot:
+		return res, NewTeamDeletedError()
```

Comment:
Are there cases when we want to load deleted team? Like when you do said deletion and client wants to replay chain to be sure it's fine (I lost track if you implemented that or no, though).

Reply:
Good point. That precheck isn’t run for deleting. I think you wouldn’t want to get a valid Team object for a deleted team because you couldn’t do anything with it. So it works out this way